packages/pluggableWidgets/carousel-native/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Fixed
+
+-   console errors were removed.
```

Comment:
Can you update the first letter of the sentence to uppercase?

Reply:
fixed